integration-tests/opentelemetry/src/main/resources/application.properties

```diff
@@ -6,5 +6,7 @@ quarkus.application.version=999-SNAPSHOT
 quarkus.otel.bsp.schedule.delay=100
 quarkus.otel.bsp.export.timeout=5s
 
+quarkus.otel.traces.eusp.enabled=true
```

Comment:
We also need a disabled test.
Probably a configuration profile will work best.

Reply:
Done, I added a profile where eusp is enabled and that's used only for my tests.